src/SDKs/SqlManagement/Sql.Tests/Utilities/SqlManagementTestUtilities.cs

```diff
@@ -26,24 +26,74 @@ namespace Sql.Tests
 {
     public class SqlManagementTestUtilities
     {
-        public const string DefaultLocationId = "japaneast";
+        private static readonly TestEnvironment _environment = new TestEnvironment(Environment.GetEnvironmentVariable("TEST_CSM_ORGID_AUTHENTICATION"));
 
-        public const string DefaultLocation =  "Japan East";
+        // We now load default locations from environment variable.
+        // Tests were recorded with this appended to DefaultLocationId=japaneast;DefaultLocation=Japan East;DefaultSecondaryLocationId=centralus;DefaultSecondaryLocation=Central US;DefaultStagePrimaryLocation=North Europe;DefaultStageSecondaryLocation=SouthEast Asia;DefaultEuapPrimaryLocation=East US 2 EUAP;DefaultEuapPrimaryLocationId=eastus2euap
+        public static string DefaultLocation
+        {
+            get
+            {
+                return _environment.ConnectionString.KeyValuePairs["DefaultLocation"];
+            }
+        }
+        public static string DefaultLocationId
+        {
+            get
+            {
+                return _environment.ConnectionString.KeyValuePairs["DefaultLocationId"];
+            }
+        }
 
-        public const string DefaultSecondaryLocationId = "centralus";
+        public static string DefaultSecondaryLocationId
+        {
+            get
+            {
+                return _environment.ConnectionString.KeyValuePairs["DefaultSecondaryLocationId"];
+            }
+        }
 
-        public const string DefaultSecondaryLocation = "Central US";
+        public static string DefaultSecondaryLocation
+        {
+            get
+            {
+                return _environment.ConnectionString.KeyValuePairs["DefaultSecondaryLocation"];
+            }
+        }
 
-        public const string DefaultStagePrimaryLocation = "North Europe";
+        public static string DefaultStagePrimaryLocation
+        {
+            get
+            {
+                return _environment.ConnectionString.KeyValuePairs["DefaultStagePrimaryLocation"];
```

Comment:
can this default to DefaultPrimaryLocation if it's not in the connection string? that way there are not so many required connection string parameters.